pkg/config/signer.go

```diff
@@ -76,3 +82,145 @@ func toLowS(key ecdsa.PublicKey, sig ecdsaSignature) ecdsaSignature {
 
 	return sig
 }
+
+// SignConfigUpdate signs the given configuration update with a
```

Comment:
nit: can probably reword this now since there isnt a "specified" signing identity passed in anymore.